<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LogViewerClass</class>
 <widget class="QMainWindow" name="LogViewerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>652</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LogViewer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="refreshButton">
          <property name="text">
           <string>Refresh</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="logComboBox"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <layout class="QFormLayout" name="formLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="fromLabel">
              <property name="text">
               <string>from</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QDateTimeEdit" name="fromTimeEdit"/>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="toLabel">
              <property name="text">
               <string>to</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QDateTimeEdit" name="toTimeEdit"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QGridLayout" name="gridLayout">
            <item row="1" column="0">
             <widget class="QCheckBox" name="warningCheckBox">
              <property name="text">
               <string>Warning</string>
              </property>
              <property name="tristate">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="0" column="2" rowspan="2">
             <widget class="QCheckBox" name="traceCheckBox">
              <property name="text">
               <string>Trace</string>
              </property>
              <property name="tristate">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QCheckBox" name="debugCheckBox">
              <property name="text">
               <string>Debug</string>
              </property>
              <property name="tristate">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QCheckBox" name="infoCheckBox">
              <property name="text">
               <string>Info</string>
              </property>
              <property name="tristate">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QCheckBox" name="errorCheckBox">
              <property name="text">
               <string>Error</string>
              </property>
              <property name="tristate">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableWidget" name="tableWidget">
      <property name="sortingEnabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="LogViewer.qrc"/>
 </resources>
 <connections/>
</ui>
